glib/src/auto/enums.rs

```diff
@@ -2352,49 +2352,3 @@ impl FromGlib<ffi::GVariantClass> for VariantClass {
         }
     }
 }
-
-#[cfg(windows)]
-#[cfg_attr(docsrs, doc(cfg(windows)))]
-#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
-#[non_exhaustive]
-#[doc(alias = "GWin32OSType")]
-pub enum Win32OSType {
```

Comment:
Moved to a different file but unfortunately also changed the name to `OSType`. Needs renaming back for 0.19 at least.

Reply:
That's actually all not great. They're not available via the `win32` module but re-exported from `lib.rs`. Fixing that in master first now...

Reply:
See https://github.com/gtk-rs/gtk-rs-core/pull/1372 . Will pull this in here after it's merged.